dist/formats/local_transaction/frontend/schema.json

```diff
@@ -349,15 +349,64 @@
         "need_to_know": {
           "$ref": "#/definitions/body_html_and_govspeak"
         },
+        "northern_ireland_availability": {
```

Comment:
Are there separate "availability" values because it's possible to have more than one? Is it possible to have  a list of "devolved_administration_availability" entries in the content item, that have an enum for the country?

Reply:
yeah there are more than one, it could be all 3.